### ISSUE TITLE
Support undocumented x86 instructions in RzIL

### ISSUE THREAD

**Issue by DMaroo** [rizinorg/rizin#4045]
x86 has some undocumented instructions. Capstone [supports](https://www.capstone-engine.org/beyond_llvm.html) disassembling some undocumented instructions. QEMU has [implemented](https://github.com/qemu/qemu/blob/80f1709aa0eb4de09b4240563463f991a5b9d855/target/i386/tcg/translate.c#L4817-L4819) some of these. It'd be nice to have these instructions lifted to RzIL once the rest of the x86 lifting is fairly complete.

Should be done once #3865 is complete.
